ext/zip/lib/mkstemp.c:75:8: warning: implicit declaration of function ‘getpid’ [-Wimplicit-function-declaration]

--- origsrc/php-7.1.9/ext/zip/lib/mkstemp.c	2017-08-30 11:13:17.000000000 -0500
+++ src/php-7.1.9/ext/zip/lib/mkstemp.c	2017-09-26 23:14:51.236035700 -0500
@@ -43,6 +43,7 @@
 #endif
 #include <stdio.h>
 #include <stdlib.h>
+#include <unistd.h>
 
 #ifndef O_BINARY
 #define O_BINARY 0
